### PR TITLE
Support usage of IBM Cloud services

### PR DESCRIPTION
**Pull request by Prajyot-Parab**
Signed-off-by: Prajyot Parab <Prajyot.Parab1@ibm.com>

### CODE REVIEW

File: openshift-install-powervs [hunk 1]
```diff
@@ -1080,6 +1080,39 @@ function variables {
...
+	  error "IBM Cloud VPC subnet name must be non-empty for using IBM Cloud Services"
+	fi
+
+	debug_switch
+	question "Enter IBM Cloud Classic username"
+	if [[ "${value}" != "" ]]; then
+	  echo "iaas_classic_username = \"${value}\"" >> $VAR_TEMPLATE
+	  question "Enter IBM Cloud Classic API Key for above username" "-sensitive"
+	  if [[ "${value}" != "" ]]; then
+		echo "iaas_classic_api_key = \"${value}\"" >> $VAR_TEMPLATE
```
Comment (bpradipt) [inline openshift-install-powervs:1105]: Can we also take the Classic API Key from the environment variable like the existing approach ?
Also I was wondering do we really need the classic username and the key both ?

cc @yussufsh

Comment (Prajyot-Parab) [reply openshift-install-powervs:1105]: Yes @bpradipt will update for environment variable I/p (possibly will need some additional checks)

**Review by bpradipt — APPROVED** [review]
/lgtm

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**